paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -84,7 +85,7 @@ internal class PaymentOptionsViewModel @Inject constructor(
     // and how to populate that view.
     override var newPaymentSelection = args.state.newPaymentSelection
 
-    override var linkInlineSelection = MutableLiveData<PaymentSelection.New.LinkInline?>(
+    override var linkInlineSelection = MutableStateFlow(
```

Comment:
No need to do, but for reference - it's a good practice to have the mutable flow as a private backing property, and expose read-only stateFlows:

```kotlin
var _linkInlineSelection = MutableStateFlow()
val linkInlineSelection: StateFlow<...> = _linkInlineSelection
```